src/DatePicker/createCalendar.js

```diff
@@ -1,5 +1,4 @@
 import flatpickr from "flatpickr";
-import l10n from "flatpickr/dist/l10n/index.js";
```

Comment:
Please restore this line. Let's discuss a flatpickr strategy in a separate issue.

Reply:
Sorry, I messed up again. I'll try again. Sorry for spamming with stupid pull requests.